lib/devise/multi_email/parent_model_extensions.rb

```diff
@@ -9,8 +9,12 @@ module ParentModelExtensions
 
       included do
         multi_email_association.include_module(EmailModelExtensions)
+
+        # Enable "autosave" to ensure email record changes are persisted
+        multi_email_association.reflection.options.merge!(autosave: true)
```

Comment:
@allenwq I found that changes to `Email` records or adding new ones for some reason was not saving when saving the `User`. If you disable this line, does it save the `Email` records for you as expected?